plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -72,6 +72,15 @@ func validPod(name string, numContainers int, resources api.ResourceRequirements
 	return pod
 }
 
+func TestPrettyPrint(t *testing.T) {
+	input := getResourceList("100m", "5Gi")
```

Comment:
Technically storage is also a resource, how about cover the following permutations:

* single resource only
* two resources
* ALL THREE RESOURCES